local/Dockerfile-postgres-inited

```diff
@@ -0,0 +1,8 @@
+FROM postgres:9.6-alpine
```

Comment:
maybe we should start testing support for postgres 13

Reply:
Indeed, 9.6 reached EOL last month. https://www.postgresql.org/support/versioning/

Reply:
I tested with both 13.5 and 14.1 – both work without an issue, but I'll handle that in a separate PR